docs/Project-Configuration/Project-Commands.md

```diff
@@ -389,7 +389,14 @@ This command will also send an external ID in the form
 Evergreen's account ID can be found on the 
```

Comment:
Historically, we advised people (and what some do) is to trust the Kernel-Build account. We no longer advise that since devprod owns other teams roles, and that would allow those teams to AssumeRole projects that trust the whole account (since their role is under our account).

Could you reword this and the segment you added to say something like "The role below is used to AssumeRole to the passed in `role_arn`. You should not trust only the account ID, as that would lead to other projects having access to your role.` and then